ibis-server/tests/routers/v2/connector/test_canner.py

```diff
@@ -0,0 +1,365 @@
+import base64
+import os
+
+import pytest
+from fastapi.testclient import TestClient
+from orjson import orjson
+
+from app.main import app
+from app.model.validator import rules
+
+"""
+The Canner Enterprise must setup below:
+- A user with PAT
+- A data source with TPCH Tiny
+- A workspace
+- A TPCH table `orders` in the workspace
+- The table `orders` must be with a comment `This is a table comment`
+- The table `orders` must have a column `o_comment` with a comment `This is a comment`
```

Comment:
```suggestion
- The table `orders` must be with a description `This is a table comment`
- The table `orders` must have a column `o_comment` with a description `This is a comment`
```
In the canner side, I think it's called `description`.